### ISSUE TITLE
Voltages of RLY1/2

### ISSUE THREAD

**Issue by mvimercati** [aver-ua/clack-ws1-meter#1]
Hi, thanks for this project, very smart!
I have clack ws1ci (Board rev 3108ci rev 7d), but my user manual reports 12V for the rly outputs (max 100mA). The whole board is power with a 12V psu.
Where did you find 15/17v? Actually I tried to directly connect a 12v relay, but I got only a smoking relay... :|

**Comment by aver-ua** [comment]
I thought so too. The valve power supply is 12 V, but in fact the voltage at the relay output is higher. Before connecting the relay, I measured the voltage with a multimeter, and got 17 volts (or so, I don't remember exactly). That's why you need a dc-dc converter...

it would be better and more correct to use a level converter on an optocoupler, but I couldn't find a ready-made suitable one

**Comment by mvimercati** [comment]
Yes, it should be the same here... actually 12V is reported in their manual, here, so I was quiet confident:
![Senza titolo](https://github.com/user-attachments/assets/e497ae4f-8930-41b8-8049-2736d6f9ea0d)

Yes now I'm looking for an optocoupler, but It's not easy to find one with a good input range (i.e 12-24v) and 3.3v output

**Comment by aver-ua** [comment]
oh wait... according to the wiring table, that You posted, it looks like your central contact of relays terminal block is not GND (negative), it is a positive! and other 2 contacts is negative, not positive. 
so, I think You need to check with a multimeter what is actually on the pins before connecting anything

**Comment by mvimercati** [comment]
I would say that you are a genious, but actually I'm a really idiot! :)
I noticed that they was inverted on the first read of the user manual, but than during the wiring of my prototype I completely forgot it! 
Thanks, now it seems to work, with direct connection without a dcdc (Actually my multimeter is broken and I'm procuring a new one).

**Comment by mvimercati** [comment]
For relay 1 i chose a different option "Softening Regen" which should count liters while in service or
during regeneration. Here the options from my UM:

* Set Time on: Relay activates after a set time at the beginning of a
regeneration cycle and then deactivates after a set period of time. The
start of regeneration is defi ned as the fi rst backwash cycle or Dn brine/Up
brine cycle, whichever comes fi rst.
* Set L Softening on: Relay activates after a set volume has been used
while in service, then deactivates after the meter stops registering fl ow and the set time period has
expired.
* Set L Softening Regen on: Relay activates after a set volume has been used while in service or
during regeneration, then deactivates after the meter stops registering fl ow and the set time period
has expired.
* Off

I just started a regeneration to see if it is true.
